docs/src/model_creation/reactionsystem_content_accessing.md

```diff
@@ -0,0 +1,249 @@
+```@example model_accessing_hierarchical
+rs.nucleus
+```
+```@example model_accessing_hierarchical
+rs.cytoplasm
+```
+!!! note
+    When accessing subsystems, we use the subsystems' [names](@ref dsl_advanced_options_naming), *not* the name of their variables (i.e. we call `rs.nucleus`, not `rs.nucleus_sys`).
+
+Next, if we wish to access a species declared as a part of one of the subsystems, we do so through it. E.g. here we access `Xₐ` (which is part of the cytoplasm subsystem):
+```@example model_accessing_hierarchical
+rs.cytoplasm.Xₐ
+```
+Note that species contained in a subsystem have the subsystem's name prepended to their name when we access it.
+
+Both subsystems contain a species `Xᵢ`. However, while they have the same name, *these are different species when accessed through their respective models*:
+```@example model_accessing_hierarchical
+isequal(rs.nucleus.Xᵢ, rs.cytoplasm.Xᵢ)
+```
+
+The same holds for the model parameters, i.e. while each subsystem contains a parameter `d`, these are considered different parameters:
+```@example model_accessing_hierarchical
+isequal(rs.nucleus.d, rs.cytoplasm.d)
+```
+The parameter `kₜ` is actually contained within the top-level model, and is accessed directly through it:
+```@example model_accessing_hierarchical
+rs.kₜ
+```
+
+Practically, when we simulate our hierarchical model, we use all of this to designate initial conditions and parameters. I.e. below we designate values for the two `Xᵢ` species and `d` parameters separately:
+```@example model_accessing_hierarchical
+using OrdinaryDiffEq, Plots
+u0 = [rs.nucleus.Xᵢ => 0.0, rs.cytoplasm.Xᵢ => 0.0, rs.cytoplasm.Xₐ => 0.0]
+ps = [rs.nucleus.p => 1.0, rs.nucleus.d => 0.2, rs.cytoplasm.kₐ => 5.0, rs.cytoplasm.d => 0.2, rs.kₜ => 0.1]
+oprob = ODEProblem(rs, u0, (0.0, 10.0), ps)
+sol = solve(oprob)
+plot(sol)
+```
+
+!!! note
+    When we access a symbolic variable through a subsystem (e.g. `rs.nucleus.Xᵢ`) that subsystem's name is prepended to the symbolic variable's name (we call this *namespacing*). This is also the case if we access it through the original model, i.e. `nucleus_sys.Xᵢ`. Namespacing is only performed when we access variables of [*incomplete systems*](@ref programmatic_CRN_construction). I.e. `isequal(nucleus_sys.d, nucleus_sys.d)` returns false (as the systems are incomplete and namespacing is performed). However, `isequal(complete(nucleus_sys).d, complete(nucleus_sys).d)` returns true  (as the systems are complete and namespacing is not performed). This is the reason that the system top-level system's name is never prepended when we do e.g. `rs.kₜ` (because here, `rs` is complete).
```

Comment:
`isequal(nucleus_sys.d, nucleus_sys.d) == true` for me. Is there a typo in the comparison you are trying to make?

Reply:
you are right, I was meaning to do
```julia
isequal(nucleus_sys.d, cytoplasm_sys.d)
```
and
```julia
isequal(complete(nucleus_sys).d, complete(cytoplasm_sys).d)
```

changed now